### PR TITLE
feat(agda): better highlight

### PR DESCRIPTION
**Pull request by Decodetalkers**
Add highlight for bool

add highlight for function

Log: better highlight

### CODE REVIEW

**Comment by Decodetalkers** [comment]
before
image

after
image

File: queries/agda/highlights.scm [hunk 1]
```diff
@@ -20,16 +20,17 @@
 ; Set
 (SetN) @type.builtin
 
+(expr . (atom) @function)
+
+((atom) @boolean
+  (#vim-match? @boolean "(true|false|True|False)$"))
```
Comment (theHamsta) [inline queries/agda/highlights.scm:26]: `(#any-of? @boolean "true" "false"  "True" "False")`

Comment (Decodetalkers) [reply queries/agda/highlights.scm:26]: > `(#any-of? @boolean "true" "false" "True" "False")`

Like this?
image

**Comment by theHamsta** [comment]
Thank you!

**Review by theHamsta — APPROVED** [review]

**Pull request merged**